pyproject.toml

```diff
@@ -70,22 +70,22 @@ python = ">=3.6,<3.9"
 boto3 = "^1.12"
 requests = "^2.23"
 matplotlib = ">=3.1,<3.4"
-attrs = ">=19.3,<20.3"
+attrs = ">=19.3,<20.4"
 jsonpickle = ">=1.3,<1.5"
 redis = "^3.4"
 numpy = "^1.16"
 scipy = "^1.4.1"
-absl-py = ">=0.9,<0.11"
+absl-py = ">=0.9,<0.12"
 apscheduler = "~3.6"
-tqdm = ">=4.31,<4.51"
+tqdm = ">=4.31,<4.55"
 networkx = ">=2.4,<2.6"
 fbmessenger = "~6.0.0"
 pykwalify = "~1.7.0"
 coloredlogs = ">=10,<15"
 "ruamel.yaml" = "^0.16.5"
 scikit-learn = ">=0.22,<0.24"
 slackclient = "^2.0.0"
-python-telegram-bot = ">=11.1,<13.0"
+python-telegram-bot = ">=11.1,<14.0"
```

Comment:
This PR https://github.com/RasaHQ/rasa/pull/6889 also replaces it. @rgstephens  are you planning to also bring your change to `2.0.x`?